ode/unit_test/Test_ODE_RK.hpp

```diff
@@ -463,6 +463,99 @@ void test_convergence_rate() {
   }
 }  // test_convergence_rate
 
+template <class execution_space>
+void test_adaptivity() {
+  using RK_type  = KokkosODE::Experimental::RK_type;
+  using vec_type = Kokkos::View<double*, execution_space>;
+  using mv_type  = Kokkos::View<double**, execution_space>;
+
+  duho my_oscillator(1, 1, 4);
```

Comment:
I've no idea what `duho` stands for; can we assume this is clear to other maintainers?

Reply:
duho is only implemented in the tests, it stands for `damped undriven harmonic oscillator`, it's actually stated where the struct is defined at the top of the file.